includes/class-sensei-quiz.php

```diff
@@ -1634,6 +1634,47 @@ public static function the_quiz_pagination() {
 
 	}
 
+	/**
+	 * Rendering html element that will be replaced with Progress Bar.
+	 *
+	 * @since 3.15.0
+	 */
+	public static function the_quiz_progress_bar() {
```

Comment:
I wonder if we could give this a slightly better name that starts with the usual verb (e.g. `output_quiz_progress_bar`).